src/components/common/notification-center.svelte

```diff
@@ -0,0 +1,51 @@
+<script>
+  import { onMount } from 'svelte';
+  import Dropdown from 'ui/dropdown.svelte';
+  import Button from 'ui/button.svelte';
+  import * as api from '@/utils/api.js';
+
+  export let notifications = null;
+  $: unread = notifications != null && notifications.some(notification => !notification.is_read);
+
+  onMount(async () => {
```

Comment:
Was it intended to load only with the page load? Maybe consider loading on dropdown opening?

Reply:
Nah. If there are new notifications, it should show a badge